### DISCUSSION TITLE
JWT public key for using web:did extension

### DISCUSSION THREAD

**Discussion by jes14** [eclipse-edc/Connector#2582] [Q&A]
Hello All,

The WEB:DID extension seems to only allow EC usage for public key JWTs. 
Found a related topic here as well https://github.com/eclipse-edc/Connector/discussions/2146. 

Can I know how to create a self-description with EC in this case or have any other solution?  

Thanks in advance.

**Comment by kkakui** [comment]
When I needed to generate a JWK (public key for JWT) for my EC public key, I found [this document included in the MVD](https://github.com/eclipse-edc/MinimumViableDataspace/blob/main/system-tests/README.md) helpful.
I hope you also find it helpful.

**Reply by jes14** [reply]
Hi @kkakui 
I was more curious about creating a self-description with the key but I appreciate your reply. Thanks! :)
